src/chat/actions.js

```diff
@@ -184,6 +184,10 @@ export const loadChats = (query) => async (dispatch, getState) => {
 }
 
 export const setTab = (tab, query) => (dispatch, getState) => {
+    dispatch({
+    	type: "LOAD_CHATS_SUCCESS",
```

Comment:
This should not emit a `LOAD_CHATS_SUCCESS` since no chats have successfully loaded. This is part of the reason the `_SUCCESS` is there at all: you can do stuff with `LOAD_CHATS_PENDING`.